styling.css

```diff
@@ -0,0 +1,143 @@
+body {
+   
+    font-family: "Palatino Linotype", Arial, sans-serif;
+    }
+
+    .customheader{
+        height: 100vh;
+        overflow: hidden;
+        position: relative;
+        background-attachment: fixed;
+        background-size: cover;
+        background-repeat: no-repeat;
+        background-position: 0 0;
+    }
+    .background-elephant {
+        background-image: url('img/elephant-279505_1920.jpg');
+    }
+
+    .background-grasshopper {
+    background-image: url('img/grasshopper.jpg');
+    }
+    .background-butterfly {
+        background-image: url('img/butterfly.jpg');
+        }
+
+Input [type="text"] {
+    Background-color:transparent;
+}
+
+
+nav {
+       
+        width: 100%;
+        /* position: fixed; */
+        background-color: brown;
+        font-size: 140%;
+    }
+
+nav ul {
+    list-style: none;
+    margin: 0;
+    padding: 20px;
+    display: flex;
+    }
+
+nav a {
+        color: coral;
+        padding: 15px;
+        text-decoration: none;
+        margin: 25px;
+    }
+
+
+h1 {
+    border-bottom: 1px solid rgb(156, 83, 83);
+    font-size:3rem;
+    font-weight: normal;
+    text-align: center;
+}
+
+h2 {
+    text-align: left;
+    padding-left: 20px;
+}
+h3 {
+    text-align: left;
+    margin-left: 20;
+    font-weight: bold;
+}
+
+/*em {
+    color: yellowgreen;
+    background-color: grey;
+}
+
+main {
+    max-width: 62rem;
+    margin:auto;
+    display: flex;
+}*/
+
+.abstand {
+    margin-left: 100px;
+}
+.svgPosition {
+    margin-left: 80px;
+}
+
+.imgPosition {
+    width:50%;
+    display: center;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+}
+.textRight{
+    text-align: right;
+    font-size: xx-large;
+    color: blanchedalmond;
+    font-weight: bold;
+}
+
+table {
+    border: thin solid black;
+    text-align: center;
+    width: 100%;  
+    background-color:#ff7950;
+    margin-top: 18rem;
+    
+}
+
+.row {
+display:flex;
+padding-inline: 80px;
+}
+.boxo {

```

Comment:
Einrückungen!
padding-inline ACHTUNG: This is an experimental technology

![grafik](https://user-images.githubusercontent.com/37736836/88664255-1b254a80-d0dd-11ea-969b-88bae9ebfbdb.png)